docs/loopback.md

```diff
@@ -0,0 +1,14 @@
+# Multiple loopback interface for tests
+
+WireMock tests require different loopback address other than 127.0.0.1.
+
+By default, in macOS, this address is not available to enable it run 
+the following command.
+
+`sudo ifconfig lo0 alias 127.0.0.2`
```

Comment:
does this need to be done for every single 127.x.x.x address?
in that case, might want to create a script to do that (eg a loop over all the 256^3 addresses)

Reply:
note: can just leave a comment about it. no need to write the script for this PR

Reply:
I'll add the script in the next commit.